Geometry/GEMGeometryBuilder/src/ME0GeometryBuilder.cc

```diff
@@ -365,16 +362,15 @@ ME0GeometryBuilder::ME0BoundPlane ME0GeometryBuilder::boundPlane(const DDFiltere
 
 // dd4hep
 
-ME0Geometry* ME0GeometryBuilder::buildGeometry(cms::DDFilteredView& fv, const cms::MuonNumbering& muonConstants) {
+ME0Geometry* ME0GeometryBuilder::buildGeometry(cms::DDFilteredView& fv, const MuonGeometryConstants& muonConstants) {
   ME0Geometry* geometry = new ME0Geometry();
+  MuonDDDNumbering mdddnum(muonConstants);
```

Comment:
`MuonDDDNumbering` shouldn't be used with DD4hep. It is old DD. Is @slomeo planning to fix this problem, or should it be fixed now?

Reply:
I shall then change it to MuonGeometryNumbering. In fact it works for both old DDD and new DD4Hep

Reply:
Please check that I have renamed MuonDDDNumbering as MuonGeometryNumbering and it works for both DDD and DD4Hep. So all geometry build should use the respective DDCompactView and MuonGeometryConstants in the ESProducer and MuonGeometryNumbering in the Builder. @slomeo Please see GEMGeometryBuilder and GEMGeometryESModule to work on RPC/CSC/DT component